src/System.Memory/dir.props

```diff
@@ -3,7 +3,6 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.1.1</AssemblyVersion>
-    <PackageVersion>4.5.1</PackageVersion>
```

Comment:
Why are we removing package version here but adding one in System.Buffers?

Reply:
System.Memory was defining PackageVersion in two different places: it's own dir.props plus inside the pkgproj. Because of the import ordering, the value that was winning was the one in the pkgproj, so I made the change there and removed this one. That said, it would have also worked if I removed the one from the pkgproj and kept this one with the right version, which in hindsight for consistency probably makes more sense. I'll go ahead and fix that.